From: Philipp Zabel <p.zabel@pengutronix.de>
Date: Fri, 13 Jun 2014 17:06:26 +0200
Subject: [PATCH] media: dt: bindings: coda: fsl,imx6dl-vpu

The i.MX6DL and i.MX6S variants use a slightly different firmware binary
than the i.MX6Q and i.MX6D. Document the missing compatible.

Signed-off-by: Philipp Zabel <p.zabel@pengutronix.de>
---
 Documentation/devicetree/bindings/media/coda.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/media/coda.txt b/Documentation/devicetree/bindings/media/coda.txt
index 90eb74cc1993..d0a2a8361081 100644
--- a/Documentation/devicetree/bindings/media/coda.txt
+++ b/Documentation/devicetree/bindings/media/coda.txt
@@ -9,7 +9,8 @@ Required properties:
   (a) "fsl,imx27-vpu" for CodaDx6 present in i.MX27
   (b) "fsl,imx51-vpu" for CodaHx4 present in i.MX51
   (c) "fsl,imx53-vpu" for CODA7541 present in i.MX53
-  (d) "fsl,imx6q-vpu" for CODA960 present in i.MX6q
+  (d) "fsl,imx6q-vpu" for CODA960 present in i.MX6Q/D
+  (e) "fsl,imx6dl-vpu" for CODA960 present in i.MX6DL/S
 - reg: should be register base and length as documented in the
   SoC reference manual
 - interrupts : Should contain the VPU interrupt. For CODA960,
